extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -71,10 +99,84 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
         // Ignore messages from other iframes.
         return;
       }
-      // Player wants to close because the user interracted on its close button
-      if (getData(e) === 'cnx_close') {
-        this.destroyPlayerFrame_();
-        this.attemptCollapse();
+      const dataString = getData(e);
+      const dataJSON = tryParseJson(dataString) || dict();
+
+      if (dataJSON['event'] === 'command') {
+        switch (dataJSON['func']) {
+          // Player wants to close because the user interracted on its close button
+          case 'cnxClose': {
```

Comment:
Same here, `cnxClose` vs `cnx_close` , will you support both for backward compatibility?

Reply:
cnx_close was not handled on the player side so it won't be breaking changes